src/main/resources/static/css/box.css

```diff
@@ -8,4 +8,94 @@
     display: flex;
     justify-content: center;
     margin-top: 90px;
-}
\ No newline at end of file
+}
+
+.custom-modal-margin {
+    margin-top: 150px;
```

Comment:
this seems awfully specific. 28%? please do not make that many new css classes - you can always use bootstrap utilities, i.e. for spacing https://getbootstrap.com/docs/4.0/utilities/spacing/ -> "mt-5" in your html for example. If you really need to define own spacing please use "rem" rather than pixel, because it looks at the relative size of the screen

Reply:
sometime rem doesn't give me the exact position that I want but will try to use bootstrap utilities

Reply:
Then I assume this has been deleted?